PennyPincher.API/PennyPincher/Controllers/MoneyManagementController.cs

```diff
@@ -1,76 +1,175 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using PennyPincher.Models;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
 using System.Text.Json;
 
 namespace PennyPincher.Controllers
 {
     [ApiController]
     [Route("/api/management")]
 
-    
+    // This controller is meant to model the projected cash flows you would like to acheive given incomes and expenses
+    // Model hypotheticals or goals and how they align with your current rate of spending in LogHistoryController
     public class MoneyManagementController : Controller
     {
-        //defining projected cash flow
-        static double income = 0;
-        static double expense_food = 0;
-        static double expense_living = 0;
-        static double expense_personal = 0;
+        List<CashFlowDto> CFList = CashFlowDataStore.CurrentCashFlow.CashFlowsList;
```

Comment:
What's the purpose of creating a variable reference, `List<CashflowDto> CfList = CashflowDataStore.CurrentCashFlow.CashFlowsList`, instead of directly accessing  `CashflowDataStore.CurrentCashFlow.CashFlowsList`?

Reply:
I shortened the reference to CFList for readability and convenience.
Otherwise I'd be typing `CashFlowDataStore.CurentCashflow.CashflowsList` each time I want to access it.